test/cypress/integration/consumer-tools/before-you-claim/before-you-claim-helpers.cy.js

```diff
@@ -1,4 +1,7 @@
-import retirementAPIResponse from '../../../fixtures/retirement-api.json';
```

Comment:
The reference to `retirement-api.json` is removed here, but the file needs to be deleted too, yea?